src/Renci.SshNet/ShellStream.cs

```diff
@@ -52,6 +54,19 @@ public bool DataAvailable
                 }
             }
         }
+        /// <summary>
+        /// Sends a Window Change Request via the Channel.
+        /// </summary>
+        /// <param name="columns">New screen width in # of columns</param>
+        /// <param name="rows">New screen height in # of rows</param>
+        /// <param name="width">New screen width in Pixels</param>
+        /// <param name="height">New screen height in Pixels</param>
+        /// <returns>true when change is successful, or false when channel is NOT open or the request </returns>
+        public bool WindowChangeRequest(uint columns, uint rows, uint width, uint height )
```

Comment:
Given the rationale below, we could consider having this method return **void**.
Please also remove the extra whitespace before the closing parenthesis.

Reply:
ok done. commit [ec60725]

Reply:
Should return false as there it is not necessary to thrown an exception in this case is typically of no consequence if the method does not succeed.